### DISCUSSION TITLE
Failed to find suitable terminal

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [wezterm/wezterm#3628] [Q&A] [answered]
Recently made the switch from Kitty to WezTerm and it's been a delight. One issue I've been running into is that when I attempt to launch Aerc from my launcher (rofi), it throws me an error: `Failed to find a suitable terminal`. 

Aerc runs perfectly fine if I launch it from within WezTerm itself. Can anyone help me troubleshoot this situation?

**Comment by yutkat** [comment, marked as answer]
I haven't checked it. I think it's not a wezterm issue but rofi configuration issue.

Set `$TERMINAL=/usr/bin/wezterm` or add `terminal: "wezterm";` in your rofi config.

If wezterm is in here, it will work well.
https://github.com/davatorium/rofi/blob/21ac2d1930a418053c9c29df713f97921ec9b259/script/rofi-sensible-terminal#L12

**Comment by lokesh-krishna** [comment]
Aha! That did it! Thanks a lot!
